eng/pipelines/maintenance-packages.yml

```diff
@@ -20,11 +30,18 @@ variables:
     value: ''
   - name: _BuildJobDisplayName
     value: 'Build and Test'
+  - name: _DotNetFinalVersionKind
+    value: ''
 - ${{ else }}:
   - name: _AdditionalBuildArgs
     value: '/p:Test=false'
   - name: _BuildJobDisplayName
     value: 'Build, Sign and Publish'
+  - name: _DotNetFinalVersionKind
```

Comment:
Why do we condition this on public vs internal? See L28.

Reply:
@ViktorHofer because we only sign and publish packages on the internal one. We also publish to the feeds out of the internal one.

I'm not convinced that's what we want. But regardless, can we please discuss that in a separate issue? I'd like this PR to just focus on the DotNetFinalVersionKind.

Reply:
Talked offline. If we want this build mode in public as well we should file another issue.